packages/v4/src/content/design-guidelines/components/date-picker/date-picker.md

```diff
@@ -0,0 +1,143 @@
+
+## Variations
+
+### Toolbar
+Use date pickers in a toolbar to filter dates, mainly as past timestamps.
+
+#### In toolbar attribute filter
+Date range fields in an attribute filter appear after selecting the Date range filter option which will push other content in the toolbar (in this case CTA button) to the right. The date(s) selected should display as chips under the filter.
+
+<img src="./img/In toolbar attribute filter.png" alt="Example of date picker in toolbar attribute filter" width="500"/>
+
+#### In toolbar filter group
+Use a date picker as its own date-range filter in a toolbar filter group when date-range filtering is a common action.
+Since horizontal space is limited in toolbar filter groups, use placeholder text to designate the range’s start and end dates.
+
+For more guidance on using filter groups, see our [toolbar design guidelines](/components/toolbar/design-guidelines).
+
+<img src="./img/Date picker in toolbar.png" alt="Example of date picker in toolbar filter group" width="780"/>
+
+Some use cases may call for a two-lined toolbar. If your toolbar spans two lines, include your dropdown and filters on the top line and your date pickers below.
+
+<img src="./img/In toolbar filter group vertical.png" alt="Example of date picker in toolbar filter group - two liner" width="350"/>
+
+For more information about using different types of filters, see our [filters guidelines](/guidelines/filters).
+
+##### Additional dropdown for simplified date-range filtering 
+Add an optional dropdown to date-range filters within a toolbar to allow users to filter by the most common date-range options associated with the corresponding table. 
+
+<img src="./img/Custom date range filtering.png" alt="Example of date picker in toolbar filter group - two liner" width="700"/>
+
+When a user selects an option from this additional dropdown, the date picker automatically populates with the first and last day of their specified range.
+
+<img src="./img/Custom date range - the most frequent options.png" alt="Example of date picker in toolbar filter group - two liner" width="700"/>
+
+Example:  If a user selects **Last month**  from the date-range dropdown, the date picker will display the first day (11/1/2020) and last day (11/30/2020) of that date range. 
+
+Once the date picker displays these prefilled values, users can edit each one. Upon customizing the date range, a user’s selected dropdown option will change from **Last month** to **Custom**.
+
+### Form
+Use a date picker in a form to display basic scheduling options, typically for future events. In some use cases, you may preselect and display the most common date options users can choose from based on your app such as ‘Execute command now’ or ‘Don’t repeat execution’
+
+<img src="./img/Date picker in form.png" alt="Example of a date picker in a form - with other elements" width="300"/>
```

Comment:
Can you make it so that the form appears on a card with a white background? Right now it just looks like a grey background with a grey border?

Reply:
Basically, do the same thing that you have in the image right below! aka having the form on a card/modal